### DISCUSSION TITLE
Ability to add lyrics/ karaoke mode

### DISCUSSION THREAD

**Discussion by Quoranevergetsanwersright** [MissingCore/Music#123] [Ideas]
So basically the ability to be able to paste the lyrics of a song.

**Comment by cyanChill** [comment]
There's currently no plans to support lyrics of any forms. For example, with online lyrics, there's issues such as cost and legal stuff ([Genius](https://genius.com/static/licensing) and [Spotify](https://support.spotify.com/us/artists/article/lyrics/) both have licenses to display them).

With your suggestion of local lyrics, there's a couple of issues, mainly complexity and effort/use.
- **Complexity:** We first need to consider the design and the implementation idea. For getting the lyrics, how exactly do we want to receive them - through manual input or through a file? If through a file, what format?
- **Effort/Use:** We need to consider how often people will use this feature and how many people will use this feature. On the previous point - what's the method of accepting the lyrics? From the previous idea, you'll need to source and convert the lyrics into a usable format, for each and every song that you want lyrics for. If getting lyrics onto the app is going to take a ton of effort, how often are people going to use this feature.

Personally, I won't be using this feature as I see no need for it in my use case. There's also the complaints that follow regardless of whether we implement this feature or not - ie: we want synchronized lyrics (complexity & bloat for those who don't use it), automatic lyrics (ie: from online), etc.

**Comment by albertgarcesb96** [comment]
There's no need of a thirdparty service for Lirics. MP3 and FLAC files already contains it's lirics inside it's metadata some are time synchronized mostly
